4.processing_image_based_profiles/data/blocklist/sc_blocklist.txt

```diff
@@ -65,3 +65,51 @@ Intensity_Cytoplasm_ER_MAX.Z
 Intensity_Cytoplasm_Mito_MAX.X
 Intensity_Cytoplasm_Mito_MAX.Y
 Intensity_Cytoplasm_Mito_MAX.Z
+Intensity_Cell_AGP_MAX.X_x
+Intensity_Cell_AGP_MAX.Y_x
+Intensity_Cell_AGP_MAX.Z_x
+Intensity_Cell_AGP_MAX.X_y
+Intensity_Cell_AGP_MAX.Y_y
+Intensity_Cell_AGP_MAX.Z_y
+Intensity_Cell_DNA_MAX.X_x
+Intensity_Cell_DNA_MAX.Y_x
+Intensity_Cell_DNA_MAX.Z_x
+Intensity_Cell_DNA_MAX.X_y
+Intensity_Cell_DNA_MAX.Y_y
+Intensity_Cell_DNA_MAX.Z_y
+Intensity_Cell_ER_MAX.X_x
+Intensity_Cell_ER_MAX.Y_x
+Intensity_Cell_ER_MAX.Z_x
+Intensity_Cell_ER_MAX.X_y
+Intensity_Cell_ER_MAX.Y_y
+Intensity_Cell_ER_MAX.Z_y
+Intensity_Cell_Mito_MAX.X_x
+Intensity_Cell_Mito_MAX.Y_x
+Intensity_Cell_Mito_MAX.Z_x
+Intensity_Cell_Mito_MAX.X_y
+Intensity_Cell_Mito_MAX.Y_y
+Intensity_Cell_Mito_MAX.Z_y
+Intensity_Cytoplasm_AGP_MAX.X_x
+Intensity_Cytoplasm_AGP_MAX.Y_x
+Intensity_Cytoplasm_AGP_MAX.Z_x
+Intensity_Cytoplasm_AGP_MAX.X_y
+Intensity_Cytoplasm_AGP_MAX.Y_y
+Intensity_Cytoplasm_AGP_MAX.Z_y
+Intensity_Cytoplasm_DNA_MAX.X_x
+Intensity_Cytoplasm_DNA_MAX.Y_x
+Intensity_Cytoplasm_DNA_MAX.Z_x
+Intensity_Cytoplasm_DNA_MAX.X_y
+Intensity_Cytoplasm_DNA_MAX.Y_y
+Intensity_Cytoplasm_DNA_MAX.Z_y
+Intensity_Cytoplasm_ER_MAX.X_x
+Intensity_Cytoplasm_ER_MAX.Y_x
+Intensity_Cytoplasm_ER_MAX.Z_x
+Intensity_Cytoplasm_ER_MAX.X_y
+Intensity_Cytoplasm_ER_MAX.Y_y
+Intensity_Cytoplasm_ER_MAX.Z_y
+Intensity_Cytoplasm_Mito_MAX.X_x
+Intensity_Cytoplasm_Mito_MAX.Y_x
+Intensity_Cytoplasm_Mito_MAX.Z_x
+Intensity_Cytoplasm_Mito_MAX.X_y
+Intensity_Cytoplasm_Mito_MAX.Y_y
+Intensity_Cytoplasm_Mito_MAX.Z_y
```

Comment:
If you haven't already, consider documenting the name choices to help clarify the convention and your data model.